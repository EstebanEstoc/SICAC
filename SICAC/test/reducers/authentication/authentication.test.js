import authenticationReducer from '../../../reducers/authentication/authenticationSlice'
import {
  toggleAuthFalse,
  toggleAuthTrue
} from '../../../reducers/authentication/authenticationSlice'

describe('authentication Reducer', () => {
  describe('INITIAL_STATE', () => {
    test('is correct', () => {
      const action = { type: 'dummy_action' }
      const initialState = false
      expect(authenticationReducer(undefined, action)).toEqual(initialState)
    })
  })

  describe('toggleAuthTrue', () => {
    test('must set the auth field to true', () => {
      const action = { type: toggleAuthTrue.type }

      expect(authenticationReducer(undefined, action)).toMatchSnapshot()
    })
  })
  describe('toggleAuthFalse', () => {
    test('must set the auth field to false', () => {
      const action = { type: toggleAuthFalse.type }

      expect(authenticationReducer(undefined, action)).toMatchSnapshot()
    })
  })
})
