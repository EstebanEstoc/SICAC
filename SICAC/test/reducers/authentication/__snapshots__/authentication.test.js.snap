// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authentication Reducer toggleAuthFalse must set the auth field to false 1`] = `false`;

exports[`authentication Reducer toggleAuthTrue must set the auth field to true 1`] = `true`;
